packages/babel-plugin/__tests__/__snapshots__/css.js.snap

```diff
@@ -710,7 +710,7 @@ import { css } from '@emotion/react';
 function media(...args) {
   return (
     /*#__PURE__*/
-    css(\\"@media (min-width:100px){\\",
+    css(\\"@media (min-width: 100px){\\",
```

Comment:
Why is a space added here?

Reply:
Because stylis is crazy-ass parser 🤣and it doesn't try to minify the input too hard, just some whitespace collapsing, nothing more.

It also tries to minimize information that it needs to track along the way, especially given the fact that it often doesn't yet know what it is currently parsing - is it a rule? is it a declaration? don't know - until it reaches either `{` or `;`

So what happens here is basically that whitespace needs to be preserved in css functions like `calc` (probably not all of it, but at least collapsed to a single space - although it doesn't try to collapse it here). So it just consumes anything between `(` and `)` as is - or rather between opening and closing tokens. The same `delimit` function is used for single quotes, double quotes, parenthesis, square brackets and obviously for strings we need to preserve all whitespace whatsoever.

I will probably revisit this later because we can minify this on our side, but didn't want to overcomplicate this PR.